tests/page_test.go

```diff
@@ -264,6 +264,11 @@ func TestPageEvaluateMappingError(t *testing.T) {
 			script:  "(6)",
 			wantErr: "Given expression does not evaluate to a function",
 		},
+		{
+			name:    "void",
+			script:  "",
+			wantErr: "evaluate requires a page function",
+		},
```

Comment:
Perfect 👍 I was mainly looking for this.

Would be good to get a test to make sure that this works as i demonstrated in the script:

    // As a string that is callable
    console.log(await jsHandle.evaluate("node => node.innerText"));
    // As a function
    console.log(await jsHandle.evaluate(node => node.innerText));

Reply:
Don't we already have tests for that?

https://github.com/grafana/xk6-browser/blob/d19534e927bfe19133205d1b16110973f3f86ca4/tests/page_test.go#L228-L239

I've also added a test for `SobekEmptyString`. Lastly, I ran your script, and it works.